pkg/bench/mixed.go

```diff
@@ -288,8 +301,8 @@ func (g *Mixed) Start(ctx context.Context, wait chan struct{}) (Operations, erro
 						console.Errorln(op.Err)
 					}
 					rcv <- op
-					objDone()
 					clDone()
+					objDone()
```

Comment:
What is the reason for this change?

Reply:
a clDone() should have a objDone otherwise we are leaking something here ?